### DISCUSSION TITLE
I can't see the www.cacti.net

### DISCUSSION THREAD

**Discussion by cold01water** [Cacti/cacti#5815] [General]
Currently, I cannot connect to www.cacti.net from multiple network bands in my company.

To be more specific, I cannot ping cacti.net's IP address "23.166.80.12". I can connect from home and cloud environments.

Is there some access restriction on the cacti.net side?

However, when I execute tracepath -n 23.166.80.12, "69.5.215.62" is output at the end of the route that has a response.

This seems to be the IP address of the Internet provider "brazos wifi".

"69.5.215.62" can be accessed from networks that can connect to cacti.net, but "69.5.215.62" cannot be pinged from networks that cannot connect to it.

I feel like there is some problem with "69.5.215.62",
but do you have any idea?

**Comment by cigamit** [comment]
I have no issues connecting, and I also tested from multiple servers in both AWS / Azure in multiple regions.  What IP Block are you coming from?

**Comment by cold01water** [comment]
I'm connecting from 211.5.xxx.xxx/xx

**Comment by cigamit** [comment]
I double checked the router, and your subnet isn't in any of the block lists.  We subscribe to a few block lists for spammers and such, but there is no 211.5.X.X on the list at all.

Brazos Wifi is our fiber provider in the datacenter, so that would be the hop before us in the DC.  Doing a traceroute from the router back to you, I get all the way over back into Japan to 27.85.152.166 before it stops.

<img width="624" alt="image" src="https://github.com/user-attachments/assets/e3241ef9-05a3-4af4-98c4-1edd9f2f92a8">

So I can't see anything stopping it on our end.  I assume you can't get to the forums or anything either?  https://forums.cacti.net

**Comment by cold01water** [comment]
Thank you for checking. I appreciate it.
Of course, I can't access https://forums.cacti.net either.
I can't connect to 23.166.80.12 at all.
However, I can see that there is no problem with cacti.net, so I'll close it here.

**Comment by optimuscream** [comment]
I have the same issue . We are from 202.158.50.x
Both cacti.net and forums.cacti.net is not reachable from our network.
